pyneuphonic/client.py

```diff
@@ -48,6 +49,7 @@ def __init__(
 
         self.voices = Voices(api_key=self._api_key, base_url=self._base_url)
         self.tts = TTS(api_key=self._api_key, base_url=self._base_url)
+        self.clone_voice = CloneVoices(api_key=self._api_key, base_url=self._base_url)
```

Comment:
As per the previous comment, if you move the `CloneVoices.post` function to `Voices.clone` then you can get rid of this line.